src/polymer/behavior-scanner.ts

```diff
@@ -105,15 +110,22 @@ class BehaviorVisitor implements Visitor {
       if (name in this.propertyHandlers) {
         this.propertyHandlers[name](prop.value);
       } else if (babel.isMethod(prop) || babel.isFunction(prop.value)) {
-        const method = esutil.toScannedMethod(
-            prop, this.document.sourceRangeForNode(prop)!, this.document);
-        this.currentBehavior.addMethod(method);
+        if (!babel.isMethod(prop) || (prop.kind !== 'get' && prop.kind !== 'set')) {
+          const method = esutil.toScannedMethod(
+              prop, this.document.sourceRangeForNode(prop)!, this.document);
+          this.currentBehavior.addMethod(method);
+        }
       } else {
         const property = toScannedPolymerProperty(
             prop, this.document.sourceRangeForNode(prop)!, this.document);
         this.currentBehavior.addProperty(property);
       }
     }
+
+    for (const prop of esutil.propertiesFromGettersAndSetters(methods, this.document)) {
```

Comment:
Could avoid needing to create `methods` if we expanded the type of the first argument to `propertiesFromGettersAndSetters`. After all, we filter out anything that isn't a method in the body of `propertiesFromGettersAndSetters`.

Reply:
i think i did this because `properties` is `ObjectProperty | ObjectMethod | SpreadProperty` and `SpreadProperty` isn't considered a `Method | Property`.

when i tried to `.filter(p => babel.isMethod(p))`, the types weren't picked up right so  iguess i went wrong somewhere.

Reply:
i see you mean just add `SpreadProperty`, just seemed odd at first but i will as its much easier